src/Storages/MergeTree/MergeTreeSettings.h

```diff
@@ -84,6 +84,8 @@ struct Settings;
     M(UInt64, replicated_deduplication_window_seconds, 7 * 24 * 60 * 60 /* one week */, "Similar to \"replicated_deduplication_window\", but determines old blocks by their lifetime. Hash of an inserted block will be deleted (and the block will not be deduplicated after) if it outside of one \"window\". You can set very big replicated_deduplication_window to avoid duplicating INSERTs during that period of time.", 0) \
     M(UInt64, replicated_deduplication_window_for_async_inserts, 10000, "How many last hash values of async_insert blocks should be kept in ZooKeeper (old blocks will be deleted).", 0) \
     M(UInt64, replicated_deduplication_window_seconds_for_async_inserts, 7 * 24 * 60 * 60 /* one week */, "Similar to \"replicated_deduplication_window_for_async_inserts\", but determines old blocks by their lifetime. Hash of an inserted block will be deleted (and the block will not be deduplicated after) if it outside of one \"window\". You can set very big replicated_deduplication_window to avoid duplicating INSERTs during that period of time.", 0) \
+    M(Milliseconds, async_block_ids_cache_min_update_interval_ms, 100, "minimum interval of updating async_block_ids_cache", 0) \
```

Comment:
it's unclear what update of a cache is from the user's standpoint of view. IMHO it is worth to write couple words here, and also add docs for this setting in `merge-tree-settings.md`

Reply:
Good, I will write more explaination and refine the doc

Reply:
I tried but was not able to elaborate in a few words. Besides, there are also some settings of async inserts absent in the doc. I will update the doc in the next PR...